### ISSUE TITLE
Philips Hue white and color ambiance 9290022166 not responding due to cluster 2

### ISSUE THREAD

**Issue by accelle17** [Koenkk/zigbee2mqtt#18636]
### What happened?

Z2m version: 1.32.2
Adapter: conbee2 firmware 0x26780700
Devices: 150
End devices: 106
Router: 44
Started: when I upgraded the zigbee2mqtt add-on from 1.32.0-2 to : 1.32.2-1.
Hue firmware: 1.53.3_r27175, also upgraded to 1.104.2

### What did you expect to happen?

On/off to happen and there should only be two endpoint (11 and 242) but there is now 3 which includes 2.
Tried removing the device normally and by force but endpoint 2 is still there.
I have two more hues of the same kind but I don't want to rejoin it to see if it acts the same.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/20294546/f2eba810-96e6-4a3f-b6b8-c69c2bce189f)

### How to reproduce it (minimal and precise)

1. remove device
2. allow permit join to all or specific device
3. device should rejoin (also tried using hue switch to reset it)
4. device rejoins and with endpoint 2,11, 242.
5. It even has reporting configured for endpoints 11 which doesn't exist on the other 2 devices with the same version/device model.
6. try to power on and it tries to publish it on endpoint 2 instead of 11

### Zigbee2MQTT version

1.32.2

### Adapter firmware version

0x26780700

### Adapter

conbee2

### Debug log

JOINING:

Z2M@hass.dayritcaldozo.family
Devices
Dashboard
Map
Settings
Groups
OTA
Touchlink
Logs
Extensions
English
Show only
all
Filter by text
0x0017880108aceff5
Log level
debug
Logging level
 
Info 2023-08-17 08:19:18Accepting joining not in blocklist device '0x0017880108aceff5'
Info 2023-08-17 08:19:18Device '0x0017880108aceff5' joined
Info 2023-08-17 08:19:18MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0017880108aceff5","ieee_address":"0x0017880108aceff5"},"type":"device_joined"}'
Info 2023-08-17 08:19:18MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x0017880108aceff5"},"type":"device_connected"}'
Info 2023-08-17 08:19:18MQTT publish: topic 'homeassistant/light/0x0017880108aceff5/light/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"brightness":true,"brightness_scale":254,"color_mode":true,"command_topic":"zigbee2mqtt/0x0017880108aceff5/set","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"effect":true,"effect_list":["blink","breathe","okay","channel_change","candle","fireplace","colorloop","finish_effect","stop_effect","stop_hue_effect"],"json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","max_mireds":500,"min_mireds":153,"name":"0x0017880108aceff5","schema":"json","state_topic":"zigbee2mqtt/0x0017880108aceff5","supported_color_modes":["xy","color_temp"],"unique_id":"0x0017880108aceff5_light_zigbee2mqtt"}'
Info 2023-08-17 08:19:18MQTT publish: topic 'homeassistant/sensor/0x0017880108aceff5/power_on_behavior/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 power on behavior","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
Info 2023-08-17 08:19:18MQTT publish: topic 'homeassistant/select/0x0017880108aceff5/power_on_behavior/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0x0017880108aceff5/set/power_on_behavior","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 power on behavior","options":["off","on","toggle","previous"],"state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
Info 2023-08-17 08:19:18MQTT publish: topic 'homeassistant/select/0x0017880108aceff5/effect/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0x0017880108aceff5/set/effect","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"icon":"mdi:palette","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 effect","options":["blink","breathe","okay","channel_change","candle","fireplace","colorloop","finish_effect","stop_effect","stop_hue_effect"],"unique_id":"0x0017880108aceff5_effect_zigbee2mqtt"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'homeassistant/sensor/0x0017880108aceff5/linkquality/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'homeassistant/sensor/0x0017880108aceff5/last_seen/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"device_class":"timestamp","enabled_by_default":true,"entity_category":"diagnostic","icon":"mdi:clock","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 last seen","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_last_seen_zigbee2mqtt","value_template":"{{ value_json.last_seen }}"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'homeassistant/sensor/0x0017880108aceff5/update_state/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:update","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 update state","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_update_state_zigbee2mqtt","value_template":"{{ value_json['update']['state'] }}"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'homeassistant/binary_sensor/0x0017880108aceff5/update_available/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"device_class":"update","enabled_by_default":false,"entity_category":"diagnostic","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 update available","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_update_available_zigbee2mqtt","value_template":"{{ value_json['update']['state'] == \"available\" }}"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'homeassistant/update/0x0017880108aceff5/update/config', payload '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/bridge/request/device/ota_update/update","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"device_class":"firmware","entity_picture":"https://github.com/Koenkk/zigbee2mqtt/raw/master/images/logo.png","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","latest_version_template":"{{ value_json['update']['latest_version'] }}","latest_version_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5","payload_install":"{\"id\": \"0x0017880108aceff5\"}","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_update_zigbee2mqtt","value_template":"{{ value_json['update']['installed_version'] }}"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'zigbee2mqtt/0x0017880108aceff5', payload '{"last_seen":"2023-08-17T08:19:18+08:00","linkquality":255,"power_on_behavior":null,"state":null,"update":{"installed_version":-1,"latest_version":-1,"state":null},"update_available":null}'
Info 2023-08-17 08:19:19Starting interview of '0x0017880108aceff5'
Info 2023-08-17 08:19:19MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0017880108aceff5","ieee_address":"0x0017880108aceff5","status":"started"},"type":"device_interview"}'
Info 2023-08-17 08:19:19MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x0017880108aceff5"},"type":"pairing"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/light/0x0017880108aceff5/light/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"brightness":true,"brightness_scale":254,"color_mode":true,"command_topic":"zigbee2mqtt/0x0017880108aceff5/set","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"effect":true,"effect_list":["blink","breathe","okay","channel_change","candle","fireplace","colorloop","finish_effect","stop_effect","stop_hue_effect"],"json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","max_mireds":500,"min_mireds":153,"name":"0x0017880108aceff5","schema":"json","state_topic":"zigbee2mqtt/0x0017880108aceff5","supported_color_modes":["xy","color_temp"],"unique_id":"0x0017880108aceff5_light_zigbee2mqtt"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/sensor/0x0017880108aceff5/power_on_behavior/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 power on behavior","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/select/0x0017880108aceff5/power_on_behavior/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0x0017880108aceff5/set/power_on_behavior","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"entity_category":"config","icon":"mdi:power-settings","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 power on behavior","options":["off","on","toggle","previous"],"state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_power_on_behavior_zigbee2mqtt","value_template":"{{ value_json.power_on_behavior }}"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/select/0x0017880108aceff5/effect/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/0x0017880108aceff5/set/effect","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"icon":"mdi:palette","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 effect","options":["blink","breathe","okay","channel_change","candle","fireplace","colorloop","finish_effect","stop_effect","stop_hue_effect"],"unique_id":"0x0017880108aceff5_effect_zigbee2mqtt"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/sensor/0x0017880108aceff5/linkquality/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:signal","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 linkquality","state_class":"measurement","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_linkquality_zigbee2mqtt","unit_of_measurement":"lqi","value_template":"{{ value_json.linkquality }}"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/sensor/0x0017880108aceff5/last_seen/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"device_class":"timestamp","enabled_by_default":true,"entity_category":"diagnostic","icon":"mdi:clock","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 last seen","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_last_seen_zigbee2mqtt","value_template":"{{ value_json.last_seen }}"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/sensor/0x0017880108aceff5/update_state/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"enabled_by_default":false,"entity_category":"diagnostic","icon":"mdi:update","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 update state","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_update_state_zigbee2mqtt","value_template":"{{ value_json['update']['state'] }}"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/binary_sensor/0x0017880108aceff5/update_available/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"device_class":"update","enabled_by_default":false,"entity_category":"diagnostic","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5 update available","payload_off":false,"payload_on":true,"state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_update_available_zigbee2mqtt","value_template":"{{ value_json['update']['state'] == \"available\" }}"}'
Debug 2023-08-17 08:19:19Received MQTT message on 'homeassistant/update/0x0017880108aceff5/update/config' with data '{"availability":[{"topic":"zigbee2mqtt/bridge/state"}],"command_topic":"zigbee2mqtt/bridge/request/device/ota_update/update","device":{"identifiers":["zigbee2mqtt_0x0017880108aceff5"],"manufacturer":"Philips","model":"Hue white and color ambiance E26/E27 (9290022166)","name":"0x0017880108aceff5","sw_version":"1.104.2"},"device_class":"firmware","entity_picture":"https://github.com/Koenkk/zigbee2mqtt/raw/master/images/logo.png","json_attributes_topic":"zigbee2mqtt/0x0017880108aceff5","latest_version_template":"{{ value_json['update']['latest_version'] }}","latest_version_topic":"zigbee2mqtt/0x0017880108aceff5","name":"0x0017880108aceff5","payload_install":"{\"id\": \"0x0017880108aceff5\"}","state_topic":"zigbee2mqtt/0x0017880108aceff5","unique_id":"0x0017880108aceff5_update_zigbee2mqtt","value_template":"{{ value_json['update']['installed_version'] }}"}'
Info 2023-08-17 08:19:20Successfully interviewed '0x0017880108aceff5', device has successfully been paired
Info 2023-08-17 08:19:20Device '0x0017880108aceff5' is supported, identified as: Philips Hue white and color ambiance E26/E27 (9290022166)
Info 2023-08-17 08:19:20MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"Hue white and color ambiance E26/E27","exposes":[{"features":[{"access":7,"description":"On/off state of this light","name":"state","property":"state","type":"binary","value_off":"OFF","value_on":"ON","value_toggle":"TOGGLE"},{"access":7,"description":"Brightness of this light","name":"brightness","property":"brightness","type":"numeric","value_max":254,"value_min":0},{"access":7,"description":"Color temperature of this light","name":"color_temp","presets":[{"description":"Coolest temperature supported","name":"coolest","value":153},{"description":"Cool temperature (250 mireds / 4000 Kelvin)","name":"cool","value":250},{"description":"Neutral temperature (370 mireds / 2700 Kelvin)","name":"neutral","value":370},{"description":"Warm temperature (454 mireds / 2200 Kelvin)","name":"warm","value":454},{"description":"Warmest temperature supported","name":"warmest","value":500}],"property":"color_temp","type":"numeric","unit":"mired","value_max":500,"value_min":153},{"access":7,"description":"Color temperature after cold power on of this light","name":"color_temp_startup","presets":[{"description":"Coolest temperature supported","name":"coolest","value":153},{"description":"Cool temperature (250 mireds / 4000 Kelvin)","name":"cool","value":250},{"description":"Neutral temperature (370 mireds / 2700 Kelvin)","name":"neutral","value":370},{"description":"Warm temperature (454 mireds / 2200 Kelvin)","name":"warm","value":454},{"description":"Warmest temperature supported","name":"warmest","value":500},{"description":"Restore previous color_temp on cold power on","name":"previous","value":65535}],"property":"color_temp_startup","type":"numeric","unit":"mired","value_max":500,"value_min":153},{"access":7,"description":"Color of this light in the CIE 1931 color space (x/y)","features":[{"access":7,"name":"x","property":"x","type":"numeric"},{"access":7,"name":"y","property":"y","type":"numeric"}],"name":"color_xy","property":"color","type":"composite"},{"access":7,"description":"Color of this light expressed as hue/saturation","features":[{"access":7,"name":"hue","property":"hue","type":"numeric"},{"access":7,"name":"saturation","property":"saturation","type":"numeric"}],"name":"color_hs","property":"color","type":"composite"}],"type":"light"},{"access":7,"description":"Controls the behavior when the device is powered on after power loss","name":"power_on_behavior","property":"power_on_behavior","type":"enum","values":["off","on","toggle","previous"]},{"access":2,"name":"effect","property":"effect","type":"enum","values":["blink","breathe","okay","channel_change","candle","fireplace","colorloop","finish_effect","stop_effect","stop_hue_effect"]},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"9290022166","options":[{"access":2,"description":"Controls the transition time (in seconds) of on/off, brightness, color temperature (if applicable) and color (if applicable) changes. Defaults to `0` (no transition).","name":"transition","property":"transition","type":"numeric","value_min":0},{"access":2,"description":"When enabled colors will be synced, e.g. if the light supports both color x/y and color temperature a conversion from color x/y to color temperature will be done when setting the x/y color (default true).","name":"color_sync","property":"color_sync","type":"binary","value_off":false,"value_on":true},{"access":2,"description":"State actions will also be published as 'action' when true (default false).","name":"state_action","property":"state_action","type":"binary","value_off":false,"value_on":true}],"supports_ota":true,"vendor":"Philips"},"friendly_name":"0x0017880108aceff5","ieee_address":"0x0017880108aceff5","status":"successful","supported":true},"type":"device_interview"}'
Info 2023-08-17 08:19:20Configuring '0x0017880108aceff5'
Info 2023-08-17 08:19:20MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"description":"Hue white and color ambiance E26/E27","friendly_name":"0x0017880108aceff5","model":"9290022166","supported":true,"vendor":"Philips"},"type":"pairing"}'
Debug 2023-08-17 08:19:20Received Zigbee message from '0x0017880108aceff5', type 'readResponse', cluster 'lightingColorCtrl', data '{"colorCapabilities":31}' from endpoint 11 with groupID null
Debug 2023-08-17 08:19:21Missing colorTempPhysicalMin and/or colorTempPhysicalMax for endpoint 0x0017880108aceff5!
Info 2023-08-17 08:19:21MQTT publish: topic 'zigbee2mqtt/0x0017880108aceff5', payload '{"color_mode":"xy","last_seen":"2023-08-17T08:19:19+08:00","linkquality":255,"power_on_behavior":null,"state":null,"update":{"installed_version":-1,"latest_version":-1,"state":null},"update_available":null}'
Debug 2023-08-17 08:19:21Received Zigbee message from '0x0017880108aceff5', type 'readResponse', cluster 'lightingColorCtrl', data '{"colorTempPhysicalMax":500,"colorTempPhysicalMin":153}' from endpoint 11 with groupID null
Info 2023-08-17 08:19:21MQTT publish: topic 'zigbee2mqtt/0x0017880108aceff5', payload '{"color_mode":"xy","last_seen":"2023-08-17T08:19:20+08:00","linkquality":255,"power_on_behavior":null,"state":null,"update":{"installed_version":-1,"latest_version":-1,"state":null},"update_available":null}'
Info 2023-08-17 08:19:21Successfully configured '0x0017880108aceff5'
Debug 2023-08-17 08:19:33Received MQTT message on 'zigbee2mqtt/0x0017880108aceff5/set' with data '{"state":"OFF"}'
Debug 2023-08-17 08:19:33Publishing 'set' 'state' to '0x0017880108aceff5'

POWERING ON:
Debug 2023-08-17 08:19:33Publishing 'set' 'state' to '0x0017880108aceff5'
Error 2023-08-17 08:19:44Publish 'set' 'state' to '0x0017880108aceff5' failed: 'Error: Command 0x0017880108aceff5/2 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received (210))'
Debug 2023-08-17 08:19:44Error: Command 0x0017880108aceff5/2 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received (210)) at DeconzAdapter.sendZclFrameToEndpoint (/app/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:656:23) at Request.send (/app/node_modules/zigbee-herdsman/src/controller/helpers/request.ts:81:20) at Endpoint.command (/app/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:833:28) at Object.convertSet (/app/node_modules/zigbee-herdsman-converters/src/converters/toZigbee.js:293:17) at Object.convertSet (/app/node_modules/zigbee-herdsman-converters/src/converters/toZigbee.js:960:32) at Publish.onMQTTMessage (/app/lib/extension/publish.ts:248:36)
Info 2023-08-17 08:19:44MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to '0x0017880108aceff5' failed: 'Error: Command 0x0017880108aceff5/2 genOnOff.off({}, {\"sendWhen\":\"immediate\",\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (no response received (210))'","meta":{"friendly_name":"0x0017880108aceff5"},"type":"zigbee_publish_error"}'

**Comment by Koenkk** [comment]
Pushed a fix!

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by accelle17** [comment]
Great. It is working now via HA and z2m ui. There's also that 64516 on the input cluster of endpoint 11. I am not able to bind it now too to tradfri switch. Let me test further on the binding.

**Comment by accelle17** [comment]
Binding now works to tradfri switch after I removed the entries on reporting and the coordinator bind. This was not present on my two similar hue bulbs. It was added during the rejoin of the problematic bulb. Again, thank you for the quick work.

Note: I am using z2m addon on HA supervised. I just copied the philips.js on the fix and add it as external converter (need to replace the .. with the static path (/app/node_modules/zigbee-herdsman-converters) to work.
